xdsl/dialects/builtin.py

```diff
@@ -386,18 +385,17 @@ def verify(self) -> None:
     @staticmethod
     @builder
     def from_dict(data: dict[str | StringAttr, Attribute]) -> DictionaryAttr:
-        to_add_data: dict[str, Attribute] = {}
```

Comment:
Why removing the type hint here?
I felt it was giving some information on what we would store in it (rather than reading the code below).

Reply:
That was by accident I think, I added it back in